Readme.md

```diff
@@ -1,24 +1,31 @@
 # rustfix
 
-The goal of this tool is to read and apply the suggestions made by rustc.
+[![Latest Version](https://img.shields.io/crates/v/rustfix.svg)](https://crates.io/crates/rustfix)
+[![Rust Documentation](https://docs.rs/rustfix/badge.svg)](https://docs.rs/rustfix)
+
+Rustfix is a library defining useful structures that represent fix suggestions from rustc
 
 ## Current status
 
 Currently, rustfix is split into two crates:
 
-- `rustfix`, a library for consuming and applying suggestions in the format that `rustc` outputs
-- and `cargo-fix`, a binary that works as cargo subcommand and that end users will use to fix their code.
+- `rustfix`, a library for consuming and applying suggestions in the format that `rustc` outputs (this crate)
+- `cargo-fix`, a binary that works as cargo subcommand and that end users will use to fix their code (maintained in the [cargo](https://github.com/rust-lang/cargo/blob/master/src/cargo/ops/fix.rs) repo).
```

Comment:
nit: I may link to <https://github.com/rust-lang/cargo/blob/master/src/bin/cargo/commands/fix.rs>, which is more like the entry of the entire "binary" part of `cargo-fix`.